.buildkite/x-pack/pipeline.xpack.packetbeat.yml

```diff
@@ -18,54 +17,274 @@ env:
+          mage build unitTest
+        key: "extended-win-11-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_11}"
+          machineType: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Windows 11 Unit Tests"
+
+      - label: ":windows: Windows 2019 Unit Tests"
+        command: |
+          Set-Location -Path x-pack/packetbeat
+          mage build unitTest
+        key: "extended-win-2019-unit-tests"
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_2019}"
+          machineType: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Windows 2019 Unit Tests"
+
+      - label: ":windows: Windows 10 System Tests"
+        key: "extended-win-10-system-tests"
+        skip: "skipping due to elastic/beats#38142"
+        command: |
+          Set-Location -Path x-pack/packetbeat
+          mage systemTest
+        agents:
+          provider: "gcp"
+          image: "${IMAGE_WIN_10}"
+          machineType: "${GCP_WIN_MACHINE_TYPE}"
+          disk_size: 100
+          disk_type: "pd-ssd"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Windows 2022 System Tests"
+
+  - group: "x-pack/packetbeat Linux arm Extended Tests"
+    key: "x-pack-packetbeat-extended-linux-arm-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*arm.*/
+    steps:
+      - label: ":linux: Ubuntu ARM Unit Tests"
+        key: "extended-arm64-unit-test"
+        command: "cd x-pack/packetbeat && mage build unitTest"
+        if: build.env("GITHUB_PR_LABELS") =~ /.*arm.*/
+        agents:
+          provider: "aws"
+          imagePrefix: "${IMAGE_UBUNTU_ARM_64}"
+          instanceType: "${AWS_ARM_INSTANCE_TYPE}"
+        artifact_paths:
+          - "x-pack/packetbeat/build/*.xml"
+          - "x-pack/packetbeat/build/*.json"
+        notify:
+          - github_commit_status:
+              context: "x-pack/packetbeat: Ubuntu ARM Unit Tests"
+
+  - group: "x-pack/packetbeat MacOS Extended Tests"
+    key: "x-pack-packetbeat-extended-macos-tests"
+    if: build.env("BUILDKITE_PULL_REQUEST") == "false" || build.env("GITHUB_PR_LABELS") =~ /.*macos.*/
+    steps:
+      - label: ":mac: MacOS Unit Tests"
+        key: "extended-macos-unit-tests"
+        command: ".buildkite/scripts/unit_tests.sh"
```

Comment:
Can we replace it with

```
          set -euo pipefail
          source .buildkite/scripts/install_macos_tools.sh
          cd heartbeat
          mage build unitTest
```          

as done for the [other pipelines](https://github.com/elastic/beats/blob/main/.buildkite/heartbeat/heartbeat-pipeline.yml#L150)? Once done, we'll be able to get rid of `unit_tests.sh` in a future PR.

Reply:
I think this hasn't been addressed and as a result the corresponding job is failing: https://buildkite.com/elastic/beats-xpack-packetbeat/builds/2826#018f065e-2eb3-42d8-9b00-58755e013db4/113-114